core/app/models/spree/shipping_manifest.rb

```diff
@@ -7,7 +7,7 @@ def initialize(inventory_units:)
 
   def for_order(order)
```

Comment:
This whole method needs to get nuked, because core doesn't create cartons from different orders. Separate PR of course